ova/generate_ova.sh

```diff
@@ -122,57 +125,61 @@ main() {
             help 0
         ;;
 
-        "-b" | "--build")
-            BUILD=true
-            shift 1
-        ;;
-
         "-v" | "--version")
             if [ -n "$2" ]; then
-                export OVA_WAZUH_VERSION="$2"
-                WAZUH_VERSION="$2"
+
+                export WAZUH_VERSION="$2"
                 HAVE_VERSION=true
             else
                 echo "ERROR Need wazuh version."
                 help 1
             fi
             shift 2
         ;;
+        "-o" | "--opendistro")
+            if [ -n "$2" ]; then
 
-        "-e" | "--elastic-version")
+                export OPENDISTRO_VERSION="$2"
+                HAVE_OPENDISTRO_VERSION=true
+            else
+                echo "ERROR Need opendistro version."
+                help 1
+            fi
+            shift 2
+        ;;
+
+        "-f" | "--filebeat")
             if [ -n "$2" ]; then
-                export OVA_ELK_VERSION="$2"
-                ELK_VERSION="$2"
+
+                export ELK_VERSION="$2"
                 HAVE_ELK_VERSION=true
             else
-                echo "ERROR: Need elastic version."
+                echo "ERROR: Need filebeat version."
                 help 1
             fi
             shift 2
         ;;
 
         "-r" | "--repository")
             if [ -n "$2" ]; then
-                export STATUS_PACKAGES="$2"
-                STATUS="$2"
+
+                export STATUS="$2"
                 HAVE_STATUS=true
             else
                 echo "ERROR: package repository is needed_."
                 help 1
             fi
             shift 2
         ;;
-
-        "-d" | "--directory")
+        "-b"|"--branch")
             if [ -n "$2" ]; then
-                export DIRECTORY="$2"
+            
+                BRANCH="$2"
+                shift 2
             else
-                echo "ERROR: Need directory to build."
                 help 1
```

Comment:
Please, add error messages.

Reply:
Done in  039d5eee2611844622e77f70f8d01bfbf9221dcb